content/tutorials/wallets.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Wallets for Ocean Tokens
+description: Learn about wallets that can hold Ocean Tokens.
+---
+
+If you want to use the Ocean Network, you need Ocean Tokens, and to get Ocean Tokens, you need to get a wallet that can hold Ocean Tokens.
+
+## Some Terminology
+
+There's a lot of terminology around wallets, so we'll start by going over some of it.
+
+A wallet might contain only one **account** (Ethereum account) or it might contain many accounts.
+
+Each account has a **balance** (e.g. 1.832 Ocean Tokens), an **address** (e.g. 0xa0A9d7e78bF29351997cA5589A0Af689eEC99211), a **public key** and a **private key**.
+
+An account is identified by its address, so if you want someone to send some Ocean Tokens to a specific account, you give them the account's address.
+
+The private key is used to spend the Ocean Tokens in the account. You must keep it secret, because anyone with that private key can spend those tokens. If you lose the secret key, and nobody else has it, then _nobody_ can spend those tokens, so don't lose it!
+
+> Note: The same account might have an Ocean Token balance, an [Ether](https://www.ethereum.org/ether) balance, and other balances.
+
+## Wallet Options
+
+Ocean Tokens are [ERC-20 tokens](https://en.wikipedia.org/wiki/ERC-20), so any wallet that supports arbitrary ERC-20 tokens should work to hold Ocean Tokens.
+
+There are many kinds of wallets (e.g. paper wallets, hardware wallets, software wallets, custodial wallets), each with its own advantages and disadvantages. There is a tradeoff between security and convenience. We encourage you to search around and read about wallets to understand the options.
+
+**We don't recommend or endorse any particular wallets at this time.** Some wallets which _might_ work with Ocean Tokens are:
+
+- Gnosis Safe
+- MetaMask (in combination with other tools such as MyEtherWallet)
```

Comment:
Metamask is not dependent on MEW. They are 2 seperate wallets. In case you wanna differentiate them, then if you are using tokens frequently, Metamask is more convenient (because of chrome plugin) else if you just want to store token in soft wallet for some short or medium term, MEW is better choice.

Reply:
@kremalicious I looked into Exodus; they have a list of ERC-20 tokens that they support, and since Ocean Tokens don't exist yet, they aren't supported yet. Exodus _can_ manage ERC-20 tokens not on their list, but the user experience is much worse.

MEW and Balance.io seem to be in a different category. They hook to actual wallets and seem to be all about giving a nicer user experience around managing tokens etc.